app/assets/stylesheets/_definitions.scss

```diff
@@ -43,6 +43,7 @@ $m-success-notification: #7AB29B;
 $m-success-notification-border: 5px solid #396554;
 $m-error-notification: #D5455A;
 $m-error-notification-border: 5px solid #6A232D;
+$m-grey-email-color: #848D95;
```

Comment:
"grey" is gramatically correct, but all other similar colors use "gray", please follow the pattern

Reply:
changed to : "gray"